.test-infra/tools/python_installer.sh

```diff
@@ -20,7 +20,7 @@
 set -euo pipefail
 
 # Variable containing the python versions to install
-python_versions_arr=("3.8.9" "3.6.13" "3.7.10" "3.9.4" "3.10.1")
+python_versions_arr=("3.7.16" "3.8.16" "3.9.16" "3.10.10")
```

Comment:
whichever version is in the beginning will be the default Python intepreter (e.g. when one runs `python3`). This line will change default from 3.8 to 3.7. Let's keep it 3.8 or change to 3.9 / 3.10.

Reply:
Changed it. PTAL @tvalentyn